### PR TITLE
Improve UI

### PR DESCRIPTION
**Pull request by aviatesk**
Please refer to https://github.com/JunoLab/atom-julia-client/pull/562

### CODE REVIEW

File: lib/debugger/breakpoints.js [hunk 1]
```diff
@@ -243,9 +245,47 @@ export default class BreakpointManager {
     this.emitter.emit('update')
   }
 
+  toggleConditionAtSourceLocation ({file, line}) {
+    const activePane = atom.workspace.getActivePane()
+
+    let found = false
+    this.fileBreakpoints.forEach(bp => {
+      if (bp.file === file && bp.line === line) {
```
Comment (pfitzseb) [inline lib/debugger/breakpoints.js]: IMHO 
``` 
if (bp.file !== file && bp.line !== line) return
```
reads a bit better.

**Review by pfitzseb — APPROVED** [review]
Looking good, thanks!

File: lib/debugger/breakpoints.js (hunk 1 above)
Comment (aviatesk) [reply lib/debugger/breakpoints.js]: Maybe you meant `||` instead of `&&` ?
I modified with `if (bp.file !== file || bp.line !== line) return`

Comment (pfitzseb) [reply lib/debugger/breakpoints.js]: Ah yeah, of course.

**Pull request merged**